hcx-qr-code-generator/src/main/java/org/swasth/HcxQRCodeGenerator.java

```diff
@@ -0,0 +1,79 @@
+package org.swasth;
+
+import com.google.gson.Gson;
+import com.google.zxing.BarcodeFormat;
+import com.google.zxing.MultiFormatWriter;
+import com.google.zxing.client.j2se.MatrixToImageWriter;
+import com.google.zxing.common.BitMatrix;
+import org.swasth.service.EncDeCode;
+import org.swasth.utils.JWSUtils;
+
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+import java.time.LocalDateTime;
+import java.util.*;
+
+public class HcxQRCodeGenerator {
+    public static void main(String[] args) throws Exception {
+        if (args.length > 0) {
+            String json = args[0];
+            Gson gson = new Gson();
+            Map<String, Object> map = gson.fromJson(json, HashMap.class);
+            System.out.println("Map received from command line argument:");
+            generateJWSToken((Map<String, Object>) map.get("payload"), (String) map.get("signingPrivateKey"));
```

Comment:
Private key shouldn't be a parameter here. It should be a configuration to the tool to use and execute.